### ISSUE TITLE
Allow disease editing when creating a case from an event person or event sample

### ISSUE THREAD

**Issue by bernardsilenou** [SORMAS-Foundation/SORMAS-Project#2110]
### Bug Description
When an event has a disease, then the the disease of the resulting case is limited to the disease of the event
### Steps to Reproduce
- Since disease is an optional field to an event:
 1. When creating a resulting case from an even, the disease of event be copied to the disease field of the case but users should be able to edit it.
2. Just as a person can be related to many cases, contacts of different diseases, this should also apply to events. ie,  a person in an event of disease X (or no disease) should be able to be converted to a case of disease Y? What happen if this person is converted to 2 cases of different diseases?
4. 

### System Details
* Device:
* SORMAS version: 1.40
* Android version/Browser:

### Additional Information
![image](https://user-images.githubusercontent.com/12996724/81977772-d4ba6680-962a-11ea-8405-b370ae9ffcfd.png)

**Comment by MateStrysewskeSym** [comment]
@bernardsilenou I disagree that users should be able to edit the disease when creating a case from an event person because in that case, the case is clearly not resulting from this event and thus there should be no linkage between the two.

Additionally, I think we should not allow users to create cases from event persons that aren't assigned a disease because in that case, the linkage also doesn't make much sense. Instead, we should allow users to retrospectively set the case as the resulting case of the event, but that's already covered in another issue.

So in my opinion, the only thing we need to do here is disabling case creation when no disease is selected for the event.

What are your thoughts?

**Comment by bernardsilenou** [comment]
I see the same challenge in both ways:
"Instead, we should allow users to retrospectively set the case as the resulting case of the event, but that's already covered in another issue." This to me is a good way to go, the challenge i see here is: what about a situation were the disease of the event is different from the disease of the case? Two cases of different diseases can tell  you they where exposed to the same event while recording their epid/ exposure data.

What about allowing an event to have no or multiple diseases? thus permitting us to like any case to the event 
If it is not possible to allow an even to have multiple diseases, then if a case report to have a link to an event that is already recorded in the system with a different disease, should we then create another event ( same description with different event id) in order to link the case to the event? This will the falsely increase the number if events also.
I discuss with Gérard and comment latter,

**Comment by bernardsilenou** [comment]
@MateStrysewskeSym For now we can go with your solution but drop the constrain  that the event and the case disease must be be the same. In future we can think of multiple disease per event. It is possible that two different cases  with different diseases result from an event. We should allow the system to document this and it is the jib of the epidemiologist to latter decide in heir analysis to count cases with  a disease different from that of the event ad resulting from the event.

When converting an event person to a case, we fill in the disease of the event in the disease field of the case, but users can change it if needed, we still keep the resulting case_id on the event table. Will this affect the logic in db?
 Just as we choose the the source case for contacts, can we also activate this method between cases, contact and events. I think an issue exit on this already, but i cant find it.

![image](https://user-images.githubusercontent.com/12996724/82895451-86736480-9f54-11ea-816c-895dd6ef06bc.png)

**Comment by bernardsilenou** [comment]
@MateStrysewskeSym  For event the diseases must not be the same but for outbreaks (new tab to be added) it must be the same

**Comment by MateStrysewskeSym** [comment]
@bernardsilenou Alright, understood!
